bda602_hw3.py

```diff
@@ -0,0 +1,54 @@
+from pyspark.conf import SparkConf
+from pyspark.context import SparkContext
+from pyspark.ml.classification import LogisticRegression
+from pyspark.ml.feature import StandardScaler, VectorAssembler
+from pyspark.sql import SparkSession
+
+
+def main():
+    appName = "App"
+    master = "local[*]"
+    spark = (
+        SparkSession.builder.appName(appName)
+        .master(master)
+        .config(
+            "spark.jars",
+            "/mnt/c/Users/thoma/scr/PythonProjectTemplate/mariadb-java-client-3.0.8.jar",
+        )
+        .enableHiveSupport()
+        .getOrCreate()
+    )
+
+    sql = (
+        "SELECT nb.batter,nb.Hit,nb.atBat,nb.game_id,nb.local_date,"
+        "SUM(nb.Hit) AS total_h,SUM(nb.atBat) as total_ab,(SUM(nb.Hit) / SUM(nb.atBat)) AS rolling_avg"
+        "FROM new_baseball nb"
+        "JOIN new_baseball nb2"
+        "on nb.batter"
+        "where nb.local_date between nb.local_date -100 and nb2.local_date"
+        "GROUP by nb.batter,nb.local_date"
+    )

```

Comment:
Your sql code is not working for me. I am not sure but shouldn't it start with three quotation marks instead of having this mark at the beginning of every line?
         """SELECT nb.batter,nb.Hit,nb.atBat,nb.game_id,nb.local_date,"
         ...
local_date"""
 like this?